<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinInfoReader</class>
 <widget class="QDialog" name="WinInfoReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #3a3633;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>222</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="last_namelLayout">
      <item>
       <widget class="QLabel" name="lbl_last_nam">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_last_name">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="nameLayout">
      <item>
       <widget class="QLabel" name="lbl_name">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_name">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="middle_nameLayout">
      <item>
       <widget class="QLabel" name="lbl_middle_name">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_middle_name">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="birthdaylLayout">
      <item>
       <widget class="QLabel" name="lbl_birthday">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Дата рождения:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="edit_birthday">
        <property name="styleSheet">
         <string notr="true">QDateEdit
{
	border: 1px solid  #3a3633;
    padding: 1px 18px 1px 3px;
	font: 10pt &quot;Tahoma&quot;;
    min-width: 6em;
	color: white;
	background-color: #524E4B;
	selection-color: white;
	selection-background-color: #d81b1b;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="adressLayout">
      <item>
       <widget class="QLabel" name="lbl_adress">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Адрес:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_adress">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="phoneLayout">
      <item>
       <widget class="QLabel" name="lbl_phone">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Телефон:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_phone">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="idLayout">
      <item>
       <widget class="QLabel" name="lbl_id">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="info_id">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>591</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="photolLayout">
    <item>
     <widget class="QLabel" name="lbl_photo">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Фото:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_photo">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_photo">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
border-width:2px;
border-color: #d81b1b;
border-style: solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_save</tabstop>
  <tabstop>edit_last_name</tabstop>
  <tabstop>edit_name</tabstop>
  <tabstop>edit_middle_name</tabstop>
  <tabstop>edit_birthday</tabstop>
  <tabstop>edit_adress</tabstop>
  <tabstop>edit_phone</tabstop>
  <tabstop>pushButton_photo</tabstop>
  <tabstop>edit_photo</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
